hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -1688,7 +1710,7 @@ solPLProg PLProg {plp_opts = plo, plp_init = dli, plp_cpprog = CPProg { cpp_at =
           let body'' = vsep $ body' <> [illegal]
           return $
             (,) (s2t k, Aeson.String $ s2t vk_) $
-              vsep $ defns <> [solFunction vk dom' ret body'']
+              vsep $ defns <> [solFunction vkWrapped [mayMemSol wrappedDomTy <> " _a"] retWrapped body''] <> [wrapper]
```

Comment:
Can you post an example (gist) of what the generated sol looks like?

Reply:
https://gist.github.com/willghatch/da4f54ccbb635098872c7a8bdf1048cc#file-eth-api-view-stack-depth-mainserver-sol-L696

Reply:
Cool